runtime/src/lib.rs

```diff
@@ -120,17 +122,20 @@ impl Filter<Call> for BaseFilter {
 pub struct IsCallable;
 frame_support::impl_filter_stack!(IsCallable, BaseFilter, Call, is_callable);
 
+const AVERAGE_ON_INITIALIZE_WEIGHT: Perbill = Perbill::from_percent(10);
 parameter_types! {
     pub const BlockHashCount: BlockNumber = 250;
-    pub const MaximumBlockWeight: Weight = 1_000_000_000;
+    pub const MaximumBlockWeight: Weight = 2 * WEIGHT_PER_SECOND;
```

Comment:
Is this equivalent to 1/3 of the max block weight, based on a 6 secs block time right?

Reply:
yeap. This means 2 seconds of Compute time for target 6 seconds block time